Description: force llc settings to ones that are known to work
Author: Peter Michael Green <plugwash@p10link.net>

--- ghc-8.4.3+dfsg1.orig/compiler/main/SysTools.hs
+++ ghc-8.4.3+dfsg1/compiler/main/SysTools.hs
@@ -297,7 +297,7 @@ initSysTools mbMinusB
                     sPgm_ar = ar_path,
                     sPgm_ranlib = ranlib_path,
                     sPgm_lo  = (lo_prog,[]),
-                    sPgm_lc  = (lc_prog,[]),
+                    sPgm_lc  = (lc_prog,[Option "-mcpu=arm1176jzf-s"]),
                     sPgm_lcc = (lcc_prog,[]),
                     sPgm_i   = iserv_prog,
                     sOpt_L       = [],
